<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_Evaluate</class>
 <widget class="QWidget" name="task_Evaluate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeView" name="dirTreeTestcase">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="dirTreeGT">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>241</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeView" name="dirTreeResult">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>241</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEval">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>452</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTestPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Test Images Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGTPath">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Groundtruth Images Directory</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtResultPath">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Save evaluate file as...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecting Testcase Images  Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecting Groundtruth Images  Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecting Save Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNewDir">
   <property name="geometry">
    <rect>
     <x>674</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>New Directory</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
